### DISCUSSION TITLE
5.x. Better Menu Organization for 5.5 :)

### DISCUSSION THREAD

**Discussion by wmdhosting** [craftcms/cms#16065] [Ideas] [answered]
Can you add these new  idea. 
I think its simple and will greatly improve organization of many things  :)

![2024-11-10_14-34-52](https://github.com/user-attachments/assets/4b68696c-ae1c-4d63-971c-e5dfae25eae7)

**Comment by brandonkelly** [comment]
Thanks for the suggestions!

Making headings toggleable was requested by #11340. Can you ellaborate on your “link” idea? What’s the use case?

**Comment by wmdhosting** [comment]
Thx for quick response 

Now  Main menu has 1 sublevel.. 

There are for example  plugins  that have multipe links, on 1 sublevel
Putting them in main menu on first level, does not make really  sense,
And even with CP Nav if we put then  let say under Commerce on 1 sublevel 

Sublevel  Links of plugin are not visible.. 


With LINK I can put LInk to plugin pages where they better suit.. 

For example   Abandoned carts plugin  i can put  links in same place where Native carts info is.. 

Or  example Back in Stock i can put on  Products page...


So Similar info is  better connected..

And Main menu is better organized

**Comment by brandonkelly** [comment, marked as answer]
Craft 5.8.0 is out with collapsible source headings (#17226).
